qoqo_aqt/Cargo.toml

```diff
@@ -30,8 +30,8 @@ roqoqo-aqt = {version="0.6", path="../roqoqo-aqt"}
 
 qoqo_calculator = {version="1.1"}
 qoqo_calculator_pyo3 = {version="1.1", default-features=false}
-qoqo = {version="1.2", default-features=false}
-roqoqo = {version="1.2"}
+qoqo = {version="1.3.1", default-features=false}
+roqoqo = {version="1.3.1"}
```

Comment:
There'll be a version 1.3.2 shortly, so I'd recommend changing this to 1.3 so that you don't need to do another merge request